masterlist.yaml

```diff
@@ -6935,6 +6935,33 @@ plugins:
       - crc: 0xA08B7C20
         util: 'TES4Edit v4.0.3h (Hotfix 1)'
 
+  - name: 'An Unexpected Company( - Better Cities)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/41801/'
+        name: 'An Unexpected Company'
+  - name: 'An Unexpected Company.esp'
+    req: [ *SilentVoice ]
+    inc:
+      - 'A Family Problem.esp'
+      - 'LGVampire.esp'
+    msg:
+      - <<: *obsolete
+        subs: [ '[An Unexpected Company v1-2-2 PATCH (with increased follow distance)](https://www.nexusmods.com/oblivion/mods/41801/)' ]
+        condition: 'version("An Unexpected Company.esp", "1.2.2", <)'
+      - <<: *patchIncluded
+        subs: [ 'Better Cities' ]
+        condition: 'active("Better Cities Resources.esm") and not file("An Unexpected Company - Better Cities.esp")'
+      - *versionXofYorGreaterRequiredOBSEv20
+    tag: [ Graphics ]
+    clean:
+      - crc: 0xD5C1D187
+        util: 'TES4Edit v4.0.4b'
+  - name: 'An Unexpected Company - Better Cities.esp'
```

Comment:
Some rules to match suggested order.

![image](https://user-images.githubusercontent.com/1944639/149843882-da1e01cf-4af0-41e6-8598-80d24c0af8c9.png)

```suggestion
  - name: 'An Unexpected Company - Better Cities.esp'
    after: [ 'Better Cities Full.esp' ]
    req:
      - name: 'Better Cities SKINGRAD.esp'
        display: 'Better Cities Full.esp or Better Cities SKINGRAD.esp'
        condition: 'not file("Better Cities Full.esp")'
```